common/src/main/java/com/noxcrew/noxesium/network/NoxesiumPacketHandling.java

```diff
@@ -148,5 +153,36 @@ public void onStartup() {
                                 packet.entityId());
             }
         });
+
+        NoxesiumPackets.CLIENT_OPEN_LINK.addListener(this, (reference, packet, context) -> {
+            try {
```

Comment:
Can this instead trigger the vanilla code for the click URL component? Like create a component and trigger clicking it. Feels weird to copy this code over.

Reply:
Or is this different logic from how vanilla works when you click a component? The behavior seems pretty similar.

Reply:
The behaviour is different as it allows the server to define extra text to the screen